packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-sizing/spark-sizing.html

```diff
@@ -0,0 +1,204 @@
+        >
+            <div class="sizing__advanced-units-container">
+                <oui-numeric
+                    name="driver-memory-overhead"
+                    data-model="$ctrl.state.driverMemoryOverheadMb"
+                    data-min="$ctrl.minMemoryOverheadMb"
+                    data-max="60000"
+                >
+                </oui-numeric>
+            </div>
+            <span class="sizing__advanced-units">MB</span>
+        </oui-field>
+        <oui-field
+            data-label="{{:: 'data_processing_submit_job_stepper_spark_worker_cores_label' | translate }}"
+        >
+            <div class="sizing__advanced-units-container">
+                <oui-numeric
+                    name="worker-cores"
+                    data-model="$ctrl.state.workerCores"
+                    data-min="1"
+                    data-max="16"
+                >
+                </oui-numeric>
+            </div>
+            <span class="sizing__advanced-units">vCores</span>
+        </oui-field>
+        <oui-field
+            data-label="{{:: 'data_processing_submit_job_stepper_spark_worker_memory_label' | translate }}"
+            data-help-text="{{:: 'data_processing_submit_job_stepper_spark_worker_memory_description' | translate }}"
+        >
+            <div class="sizing__advanced-units-container">
+                <oui-numeric
+                    name="worker-memory"
+                    data-model="$ctrl.state.workerMemoryGb"
+                    data-min="1"
+                    data-max="60"
+                >
+                </oui-numeric>
+            </div>
+            <span class="sizing__advanced-units">GB</span>
+        </oui-field>
+        <oui-field
+            data-label="{{:: 'data_processing_submit_job_stepper_spark_worker_memory_overhead_label' | translate }}"
+        >
+            <div class="sizing__advanced-units-container">
+                <oui-numeric
+                    name="worker-memory-overhead"
+                    data-model="$ctrl.state.workerMemoryOverheadMb"
+                    data-min="$ctrl.minMemoryOverheadMb"
+                    data-max="60000"
+                >
+                </oui-numeric>
+            </div>
+            <span class="sizing__advanced-units">MB</span>
+        </oui-field>
+        <oui-field
+            data-label="{{:: 'data_processing_submit_job_stepper_spark_worker_count_label' | translate }}"
+        >
+            <oui-numeric
+                name="worker-count"
+                data-model="$ctrl.state.workerCount"
+                data-min="1"
+                data-max="10"
+            >
+            </oui-numeric>
+        </oui-field>
+        <div class="sizing__pricing">
+            {{ :: 'data_processing_submit_job_stepper_spark_estimated_price' |
+            translate }}
+            <strong
+                ><span ng-bind="$ctrl.computePrice().toFixed(3)"></span
+                >€/min</strong
```

Comment:
Well, what about customers in subsidiaries without euros ?

Even if the product is only available in France for example, this should be taken into account